gradle/libs.versions.toml

```diff
@@ -23,6 +23,8 @@ junitVersion = "4.13.2"
 junitExtVersion = "1.1.5"
 multidexVersion = "2.0.1"
 materialVersion = "1.10.0"
+kakaoversion = "3.4.1"
```

Comment:
Good day, thanks for the reporting about the issue. Hm, that is quite strange, because we migrated from 3.4.1 to 3.5.0 without any issues. And we have more than 1.3k UI tests. 

As well, could you provide any information about which methods visibility has been affected?

Unfortunately I'm on vacation and will be back at the end of December

Reply:
Hey there! I just checked the `getResourceDwarable()` function that you imported into the PR, and it's signature hasn't changed since the last 3 years. Could you please add more details on to what exactly has changed it's visibility?

Reply:
Yes, the signature hasn't changed, totally agree with that.
However, if you checkout this branch which consumes the released library you'll realise that the function is visible in `3.4.1` but not in `3.5.0`. 
I checked the source of the published library and I see the function. I'm not sure what's going on in terms of visibility
<img width="1426" alt="Screenshot 2023-12-11 at 09 44 24" src="https://github.com/KakaoCup/Kakao/assets/2768618/1f4c3f04-55cd-45b1-9c2f-cff7e1afe371">

<img width="1427" alt="Screenshot 2023-12-11 at 09 44 37" src="https://github.com/KakaoCup/Kakao/assets/2768618/6cd5b871-81c0-4982-af76-c0448a299c29">